src/styles/index.ts

```diff
@@ -1700,7 +1700,7 @@ const styles = (theme: ThemeColors) =>
 
         createMenuContainer: {
             width: variables.sideBarWidth - 40,
-            paddingVertical: 12,
+            paddingVertical: 16,
```

Comment:
Instead of writing 16, what if we wrote `variables.componentBorderRadiusLarge`? Just an idea. I can see how it might be confusing, but technically we want this top padding to be the same as our border radius here.